Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.py

```diff
@@ -0,0 +1,2214 @@
+                                        headers=['id', 'type_str', 'status_str', 'result_str', 'start_ts', 'end_ts'],
+                                        headerTransform=string_to_table_header),
+        raw_response=raw_response
+    )
+
+
+def list_tags_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Command to list all tags
+        Args:
+            client: Client object with request
+            args: demisto.args()
+
+        Returns:
+            Outputs.
+        """
+
+    query_params = {
+        'folder': encode_string_results(args.get('folder'))
+    }
+    tsg_id = args.get('tsg_id')
+    if tag_id := args.get('tag_id'):
+        raw_response = client.get_tag_by_id(query_params=query_params, tag_id=tag_id, tsg_id=tsg_id)
+        outputs = raw_response
+    else:
+        query_params.update(get_pagination_params(args))
+
+        raw_response = client.list_tags(query_params=query_params, tsg_id=tsg_id)  # type: ignore
+        outputs = raw_response.get('data', [])
+
+    return CommandResults(
+        outputs_prefix=f'{PA_OUTPUT_PREFIX}Tag',
+        outputs_key_field='id',
+        outputs=outputs,
+        readable_output=tableToMarkdown('Tags',
+                                        outputs,
+                                        headers=['id', 'name', 'folder', 'color', 'comments'],
+                                        headerTransform=string_to_table_header),
+        raw_response=raw_response
+    )
+
+
+def create_tag_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Command to create new tag
+    Args:
+        client: Client object with request
+        args: demisto.args()
+
+    Returns:
+        Outputs.
+    """
+
+    tag = {
+        'name': args.get('name')
+    }
+
+    if color := args.get('color'):
+        tag['color'] = color
+
+    if comments := args.get('comments'):
+        tag['comments'] = comments
+
+    query_params = {'folder': args.get('folder')}
+    tsg_id = args.get('tsg_id')
+
+    demisto.debug(f'Sending tag to the API. Tag: {tag}')
+
+    raw_response = client.create_tag(query_params=query_params, tag=tag, tsg_id=tsg_id)  # type: ignore
+
+    return CommandResults(
+        outputs_prefix=f'{PA_OUTPUT_PREFIX}Tag',
+        outputs_key_field='id',
+        outputs=raw_response,
+        readable_output=tableToMarkdown('Address Object Created', raw_response, headerTransform=string_to_table_header),
+        raw_response=raw_response
+    )
+
+
+def update_tag_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Command to update an existing tag
+    Args:
+        client: Client object with request
+        args: demisto.args()
+
+    Returns:
+        Outputs.
+    """
+
+    query_params = {
+        'folder': encode_string_results(args.get('folder'))
+    }
+    # first get the original tag, so user won't need to send all data
+    tag_id = args.get('tag_id', '')
+    tsg_id = args.get('tsg_id')
```

Comment:
?

Reply:
just look similar. one is the tag and the other is the tsg (the tenant)